src/test/java/StorageTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class StorageTest {
+    @Test
+    public void emptyFileTest() {
+        Storage store = new Storage("data/new.txt");
+        ArrayList<Task> ls = store.loadFile();
+        assertEquals(new ArrayList<>(), ls);
+    }
+
+    @Test
+    public void saveAndLoadTest() {
```

Comment:
You can add more description to the name using three part format. (Under Naming in coding standard)